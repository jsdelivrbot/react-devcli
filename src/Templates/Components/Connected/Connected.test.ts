import {statefulJS, statelessJS, statefulTS, statelessTS} from './Connected';

describe('Connected component template test suite', (): void => {
    test('should match stateful connected component template snapshot for JS language', (): void => {
        expect(statefulJS()).toMatchSnapshot();
    });

    test('should match stateless connected component template snapshot for JS language', (): void => {
        expect(statelessJS()).toMatchSnapshot();
    });

    test('should match stateful connected component template snapshot for TS language', (): void => {
        expect(statefulTS()).toMatchSnapshot();
    });

    test('should match stateless connected component template snapshot for TS language', (): void => {
        expect(statelessTS()).toMatchSnapshot();
    });
});
