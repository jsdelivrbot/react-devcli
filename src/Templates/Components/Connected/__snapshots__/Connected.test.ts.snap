// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Connected component template test suite should match stateful connected component template snapshot for JS language 1`] = `
"import React from 'react';

export class REACT__COMPONENT__NAME extends React.Component {
   constructor(props) {
       super(props);
   }

   render() {
       const Component = this.props.component;

       return (
           <Component {...this.props}></Component>
       );
   }
};"
`;

exports[`Connected component template test suite should match stateful connected component template snapshot for TS language 1`] = `
"import React from 'react';

export class REACT__COMPONENT__NAME extends React.Component {
   constructor(props) {
       super(props);
   }

   render() {
       const Component = this.props.component;

       return (
           <Component {...this.props}></Component>
       );
   }
};"
`;

exports[`Connected component template test suite should match stateless connected component template snapshot for JS language 1`] = `
"import React from 'react';

export class REACT__COMPONENT__NAME extends React.Component {
   constructor(props) {
       super(props);
   }

   render() {
       const Component = this.props.component;

       return (
           <Component {...this.props}></Component>
       );
   }
};"
`;

exports[`Connected component template test suite should match stateless connected component template snapshot for TS language 1`] = `
"import React from 'react';

export class REACT__COMPONENT__NAME extends React.Component {
   constructor(props) {
       super(props);
   }

   render() {
       const Component = this.props.component;

       return (
           <Component {...this.props}></Component>
       );
   }
};"
`;
